README.md

```diff
@@ -75,13 +60,23 @@ separately as either a Docker container, VM, or Kubernetes pod.
 
 ![deployment-models](images/opencensus-service-deployment-models.png)
 
-### <a name="deploy-k8s"></a>Kubernetes
+## <a name="getting-started"></a>Getting Started
+
+### <a name="getting-started-demo"></a>Demo
+
+Instructions for setting up an end-to-end demo environment can be found [here](https://github.com/census-instrumentation/opencensus-service/tree/master/demos/trace)
 
-Coming soon!
+### <a name="getting-started-k8s"></a>Kubernetes
 
-### <a name="deploy-standalone"></a>Standalone
+Apply the [sample YAML](example/k8s.yaml) file:
 
-Create an Agent [configuration file](#configuration-file) as described above.
+```shell
+$ kubectl apply -f example/k8s.yaml
+```
+
+### <a name="getting-started-standalone"></a>Standalone
+
+Create an Agent [configuration](#config) file based on the options described below. By default, the agent has the opencensus receiver enabled, but no exporters.
```

Comment:
Unfortunately, at this moment this is not true for the agent. It requires a config file and it needs the opencensus receiver explicitly there.